src/evm/contracts/Venus.ts

```
@@ -146,10 +210,27 @@ export interface Venus extends BaseContract {
     "nonpayable"
   >;
 
+  receiveTransmissionEvent: TypedContractMethod<
+    [
+      receiptIndex: BytesLike,
+      receiptRLPEncodedBytes: BytesLike,
+      logIndex: BigNumberish,
+      proof: BytesLike
+    ],
+    [void],
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)